tee-worker/enclave-runtime/src/attestation.rs

```diff
@@ -142,7 +142,8 @@ pub unsafe extern "C" fn generate_ias_ra_extrinsic(
 	}
 	let mut url_slice = slice::from_raw_parts(w_url, w_url_size as usize);
 	let url = match String::decode(&mut url_slice) {
```

Comment:
Wasn't it previously decoded (`String`) , not encoded (`Vec<u8>`) ? 😕 
Your comment says like there was no decoding performed.